src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -304,60 +323,91 @@ internal static async Task CheckForUpdates()
         private static async Task<Release> QueryNewReleaseAsync(SemanticVersion baselineVersion)
         {
             bool isStableRelease = string.IsNullOrEmpty(PSVersionInfo.PSCurrentVersion.PreReleaseLabel);
-            string queryUri = isStableRelease ? LatestReleaseUri : Last3ReleasesUri;
+            string[] queryUris = s_notificationType switch
+            {
+                NotificationType.LTS => new[] { "https://pscoretestdata.blob.core.windows.net/buildinfo/lts.json" },
+                NotificationType.Default => isStableRelease
+                    ? new[] { "https://pscoretestdata.blob.core.windows.net/buildinfo/stable.json" }
+                    : new[] { "https://pscoretestdata.blob.core.windows.net/buildinfo/stable.json", "https://pscoretestdata.blob.core.windows.net/buildinfo/preview.json" },
```

Comment:
Why do we prefer static fields for consts like `DoneFileNameTemplate` but not here?

Reply:
No reason :) I will make them const strings.